components/servo/Cargo.toml

```diff
@@ -78,6 +78,7 @@ webrender_api = {git = "https://github.com/servo/webrender", features = ["ipc"]}
 webdriver_server = {path = "../webdriver_server", optional = true}
 webvr = {path = "../webvr"}
 webvr_traits = {path = "../webvr_traits"}
+webxr-api = "0.0.1"
```

Comment:
Can we use git deps for now? I don't think we're at a stage yet where we should be using versioned crates.

Reply:
Ok